docassemble/ALAutomatedTestingTests/data/questions/all_tests.yml

```
@@ -31,6 +31,7 @@ code: |
   see_simple_doc
   
   button_event_action
+  simple_dropdown_field
```

I think the point of the test is that it's focused on containing only dropdown fields. I want to give it a name that will help avoid future people adding other things to this page. You can batch these changes in the "files" github tab of this PR.

```suggestion
  single_dropdown_field
```